### DISCUSSION TITLE
Clicking anywhere allows mouse to leave screen

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#18327] [Q&A] [answered]
https://www.youtube.com/shorts/1uqx_UNxIPo

I cannot recreate this in earlier versions of osu!lazer, just 2022.515.0
logs not necessary? 
video is in fullscreen, not borderless. 

arch linux

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/18330
